### PR TITLE
add changelog and release notes for LA

### PR DESCRIPTION
**Pull request by megastef**

### CODE REVIEW

File: docs/logagent/releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+title: Logagent Release Notes
+description: Features and improvements releases for Sematext Logagent
+
+Logagent is a dynamic project, and releases are typically in a high frequency.
```
Comment (otisg) [inline docs/logagent/releasenotes.md]: Logagent is a dynamic project, and releases are typically in a high frequency. => Logagent is a dynamic project with frequent releases.

File: docs/logagent/releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+title: Logagent Release Notes
+description: Features and improvements releases for Sematext Logagent
+
+Logagent is a dynamic project, and releases are typically in a high frequency. 
+We list here just important changes, new features, and critical bug fixes.
```
Comment (otisg) [inline docs/logagent/releasenotes.md]: We list here just important changes, new features, and critical bug fixes.  => This Changelog contains only important changes, new features, and critical bug fixes.

File: docs/logagent/releasenotes.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+### New Features
+
+- RedHat container certification
+- Kubernetes audit log plugin to receive data from Kubernetes audit log webhook
+  
+### Improvements
+
+- Docker images: 
+    - Add health check command
+    - Dockerfile for aarch64
```
Comment (otisg) [inline docs/logagent/releasenotes.md]: aarch64?

File: docs/logagent/releasenotes.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    - Dockerfile for RedHat Enterprise
+- Creation of binary executables using nexe.
```
Comment (otisg) [inline docs/logagent/releasenotes.md]: No need for a period here and below

File: docs/logagent/releasenotes.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  Binaries can run without installing nodejs runtime.
```
Comment (otisg) [inline docs/logagent/releasenotes.md]: nodejs => NodeJS?

File: docs/logagent/releasenotes.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+    - Dockerfile for RedHat Enterprise
+- Creation of binary executables using nexe.
+  Binaries can run without installing nodejs runtime. 
+- Automated changelog and release notes
+
+
+### Bugfixes
+
+- Log patterns: Bugfix for web server log patterns
+- Docker plugin: Bugfix for not collecting container logs after quick container restart (timing issue)
```
Comment (otisg) [inline docs/logagent/releasenotes.md]: timing => timeout?

File: docs/logagent/releasenotes.md (hunk 3 above)
Comment (megastef) [reply docs/logagent/releasenotes.md]: Yes. https://en.wikichip.org/wiki/arm/aarch64

File: docs/logagent/releasenotes.md (hunk 6 above)
Comment (megastef) [reply docs/logagent/releasenotes.md]: Actually, it was a timing issue when container did restart within a few seconds. The collector did detach from the container log stream after it was already restarted.

File: docs/logagent/releasenotes.md (hunk 5 above)
Comment (megastef) [reply docs/logagent/releasenotes.md]: On nodejs.org it is called "Node.js is a JavaScript runtime". Changing to "Node.js runtime".

File: docs/logagent/releasenotes.md (hunk 1 above)
Comment (otisg) [reply docs/logagent/releasenotes.md]: Ping @megastef ?  Or am I looking at outdated version?

**Pull request merged**